polyhost/gui/fontpack_extend_dialog.py

```
@@ -0,0 +1,869 @@
+                    src, codepoint_range=(int(self._first.text(), 16),
+                                          int(self._last.text(), 16)),
+                    opts=self._options(), global_index=gidx)
+        except Exception as e:  # noqa: BLE001
+            if auto:
+                self._status.setText(f"(auto preview) {e}")
+            else:
+                QMessageBox.critical(self, "Build failed", str(e))
+            return
+        self._built = (self._bundle.currentIndex(), new)
+        self._render_preview()
+        self._status.setText(f"Built {new.glyph_count} glyph(s), U+{new.first:04X}–"
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)